src/pages/map.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import Layout from '../components/Layout';
+import styles from '../components/Navbar.module.css'
+
+import useAPIRequest from '../hooks/useAPIRequest';
+
+function Map() {
+    const [farmname, setFarmname] = useState("");
+    const [farmemail, setFarmemail] = useState("");
+    const [farmnumber, setFarmnumber] = useState("");
+    async function postFarm(e) {
+        e.preventDefault();
+        console.log("== Adding farm with these parameters:", farmname, farmemail);
+        //const res = await fetch('/api/accessBackend/https://native-plants-backend.herokuapp.com/i/INSERT INTO rev2.farms(farm_name, contact_email) VALUES (%s) /'+farmname+', '+farmeamil,{
+        const res = await fetch('/api/accessBackend', {
+            method: 'POST',
+            body: JSON.stringify( {
+                table_name: "map",
+                query_type: "INSERT",
+                query_fields: ['farm_name','contact_email'],
+                query_values: [farmname, farmemail]
+            }),
+            headers: {
+                'Content-Type': 'application/json'
+            }
+        })
+        const resBody = await res.json();
+        console.log(resBody);
+    }
+
+    return (
+        <Layout>
+        <form className={styles.containerMap}> 
+            <div>
+                <iframe
+                    id="frame-gis"
+                    src= "http://localhost:8090/mapview/face/RTmapScreen.html"
```

Comment:
I am pretty sure this will not deploy correctly, but I think the idea is translatable to something more native to the React & Next.js framework.
We may want to consider waiting to merge this until we have the frontend on an actual server, but I'll leave that up to @Hunteret7 to decide.

Reply:
As it is presented in this PR, the iframe will not load anything because nothing is being served from localhost:8090.
If you want help turning RTmapScreen.html into something "React-y" let me know and I will help. 

Example screenshot based on current commit:
![image](https://user-images.githubusercontent.com/46549109/160461741-deae21ab-c71f-4461-a3df-6097d898c3f9.png)